### ISSUE TITLE
[Documentation] Easier-to-use communication log

### ISSUE THREAD

**Issue by kategarklavs** [18F/web-design-standards#1467]
## Description

This is a subtask of #1401. 

For this task, we'd like to gather more info about how the team feels re: completing the communication log (as part of email duty); more specifically, we'd like to gather — and implement — folks' feedback on how we can make the process easier. Because we may rely on the log to track certain key metrics, we'd like to make sure it's accurately maintained.

- [ ] Interview team about how we can make the communication log easier to use
- [ ] Set up Slackbot reminder (Mondays)

**Comment by kategarklavs** [comment]
On 9/15/16, @jameshupp and I discussed our metrics-tracking plan and brainstormed some ways to update our comms log. We deprecated the previous sheet (which folks can still view — it's just labeled as deprecated), and we created a new sheet with drop-downs. The preset options will make it easier for folks to record communications from all channels, and the issue-type buckets will help us identify, at a glance, the types of content that might need work. Additionally, there's a notes field where people can list how they addressed (or plan to address) various issues.

**Comment by jameshupp** [comment]
One thought on this: do we need to track GH issues in the comms log? I think the other interaction types are definitely helpful, but a lot of our response to actionable prompts from other communication is to create or update issues. They end up right here once we've synthesized them. If someone files an issue it stays until we close it. So I guess I'm thinking we don't need to track that elsewhere alongside other comms, since this is where they come together in the end.

**Comment by kategarklavs** [comment]
Ahh, that's a good point. My only thought re: logging GH issues is that having all feedback types consolidated in a single form would make it really easy for folks to have an at-a-glance view of % of inquiries coming through each channel (GH included). (That said, I'm not precious about keeping it!)

**Comment by jameshupp** [comment]
Yeah, that's a good point. I was thinking of the log as a resource we could use to build better work, which we track in issues anyway. Inquiries that don't need work aren't really the important ones to capture, even though we will capture them. But since we want to know where to prioritize better communication, including Github, it's probably valuable to keep them all together. Over the last two months we have, I think, eleven issues opened by people who aren't team members, so it's not overwhelming to track them.

I think I'm on team keep Github now!

**Comment by kategarklavs** [comment]
Ah, right on! In that case, I'm happy with keeping GitHub, and if we find the info isn't useful, we can always modify the sheet.

**Comment by jameshupp** [comment]
One more thing I've been doing today: testing the new log with any comms since September 1. I've noticed a couple things that needed updating, and I'm taking care of those, but I think it's basically good to go.

**Comment by maya** [comment]
Looks great! I like the categories that you have so far. I noticed sometimes categories can border each other. For example: "feature request" and "customization" but I'll leave it to the discretion of people adding labels.

I think bug reports may be common enough to be their own category. 

Another one to consider, not necessarily right now is **process** or **change management**? I think right now we use "selling points" for those concerns. Not sure if we have a stance on answering those kinds of questions yet, but just something to consider.

For using the Standards, right now we have "getting started" and "customization" and I wonder if we need a "troubleshooting" or more broad "implementation" category.

**Comment by kategarklavs** [comment]
@maya Hey, thank you for your feedback — this is excellent!

Re: your point on "feature request" vs. "customization," I see what you mean. As I was creating the categories, I was envisioning "feature request" as a means of capturing fairly straightforward feature requests (For example, "Can you all add more data tables?"), and "customization" was more to capture questions about whether the Standards are customizable/how people can customize them — does this seem like a reasonable distinction? (I'm also open to new categories that might be more precise.)

+1 to change management — I like it! 

And on the topic of "troubleshooting"/"implementation" — I'm of two minds here. On the one hand, I like the idea of a broader "troubleshooting" category; on the other, it feels like that category might be too broad to direct our attention to certain pieces of content. (As an example: If folks are having difficulty understanding how to contribute, we'd file that inquiry under "Contribution," and would then know to make our contribution guidelines more specific.) Does that make sense? That said, I'm open to "troubleshooting"/"implementation" if you all feel like they'd help us capture actionable data.

**Comment by maya** [comment]
@kategarklavs Happy to leave it as is and see how the categories hold up in use! Those were just some ideas that came to mind :)

**Comment by kategarklavs** [comment]
@maya Cool cool! That works for me :) And my sense is that we'll need to keep adjusting the categories as time goes on.
